Sources/PrimerSDK/Classes/Core/PrimerHeadlessUniversalCheckout/Composable/ACH/Models/ACHAdditionalInfo.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  ACHAdditionalInfo.swift
+//  PrimerSDK
+//
+//  Created by Stefan Vrancianu on 31.05.2024.
+//
+
+import UIKit
+
+@objc public class ACHAdditionalInfo: PrimerCheckoutAdditionalInfo {}
+
+// MARK: ACH bank account collector view controller
+@objc public class ACHBankAccountCollectorAdditionalInfo: ACHAdditionalInfo {
+
+    public var collectorViewController: UIViewController
```

Comment:
The merchant needs to present this bank account collector screen. We thought of using this delegate as a means of doing so, since we already used it for displaying the MandateInfo.